content/posts/0005-dotnet-rearchitecture.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Introducing the Paketo .NET Core Buildpack v0.1.0
+date: "2021-02-09"
+slug: dotnet-core-rearchitecture
+author: fgallinajones
+---
+
+&lt;PICTURE AT THE TOP>
+
+Paketo is pleased to unveil our new and improved .NET Core Buildpack! With the
+release of [Paketo .NET Core Buildpack
+0.1.0](https://github.com/paketo-buildpacks/dotnet-core/releases), you’ll enjoy
+a host of new features and improvements that we hope will make containerizing
+your .NET apps a breeze.
+
+With the new buildpack, you can now:
+*   Build .NET 5 apps
+*   Provide your app as source code, a framework-dependent deployment (FDD), a
+    framework-dependent executable (FDE), or a self-contained deployment
```

Comment:
You could do this in the previous iteration of the buildpack as well.

Reply:
Yeah, fair point. I wasn't sure whether to include this given that we've only just started _testing_ this functionality. Happy to remove.